app/api/v1/endpoints/chat.py

```
@@ -226,11 +226,24 @@ async def chat_websocket(
+        user_id = "websocket_user"  # Default fallback
         
-        # Connect to WebSocket manager (this handles websocket.accept())
+        # Authenticate before accepting connection
+        if token:
+            try:
+                from app.core.security import verify_access_token
+                payload = verify_access_token(token)
+                if payload and payload.get("sub"):
+                    user_id = payload["sub"]
+                    logger.info("Chat WebSocket user authenticated", user_id=user_id)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)